lib/datadog/tracing/span.rb

```diff
@@ -207,6 +210,10 @@ def start_time_nano
       def duration_nano
         (duration * 1e9).to_i
       end
+
+      def top_level?
```

Comment:
Can you write some public documentation here, explaining what `top_level?` is, similar to what you did in the PR description?